app/src/main/java/org/akvo/flow/service/DataSyncService.java

```diff
@@ -88,7 +88,7 @@
  */
 public class DataSyncService extends IntentService {
 
-    private static final String TAG = "SyncService";
+    private static final String TAG = "DataSyncService";
```

Comment:
This is used in the constructor:
 public DataSyncService() {
        super(TAG);
    }